applications/nrf5340_audio/README.rst

```diff
@@ -615,6 +615,14 @@ Only one of the following :file:`.conf` files is included when building:
   When building using the command line, you must explicitly specify if :file:`prj_release.conf` is going to be included instead of :file:`prj.conf`.
   See :ref:`nrf53_audio_app_building` for details.
 
+Requirements for FOTA
+=====================
+
+To test FOTA you need an Android or iOS device with the `nRF Connect Device Manager`_ app installed.
+
+If you want to do FOTA upgrades for the application core and the network core at the same time you need an external flash shield.
```

Comment:
We must add something like this: Performing separate application and network core upgrades are suitable for testing, but for production, both cores must be updated at the same time. When updates takes place in the inter-core communication module (HCI RPMsg) communication between the cores will break if they are not updated together.

Reply:
Added this text to the 'important' part under "Configuring FOTA upgrades"